lib/dat-worker-pool/worker.rb

```diff
@@ -135,16 +140,19 @@ def dwp_setup
       def dwp_make_unavailable
         @dwp_runner.make_worker_unavailable(self)
         dwp_run_callback 'on_unavailable'
+        dwp_log "Unavailable"
       end
 
       # this is a separate method so the test runner can call it individually
       def dwp_make_available
         @dwp_runner.make_worker_available(self)
         dwp_run_callback 'on_available'
+        dwp_log "Available"
       end
 
       # this is a separate method so the test runner can call it individually
       def dwp_work(work_item)
+        dwp_log "Working"
```

Comment:
@jcredding maybe include the work item?  `dwp_log "Working, item: #{work_item.inspect}"`.  Just a thought - I'm OK either way on this.